### ISSUE TITLE
1.7 Third-person blocking pose

### ISSUE THREAD

**Issue by AndrewScripter587** [ViaVersion/ViaFabricPlus#563]
### Problem Description

When playing on a MC beta 1.8 to release 1.7.10 server, the release 1.8 third-person sword blocking pose is used, when prior MC versions to 1.8 had a different third-person blocking pose.

### Solution Description

I would like the 1.7 and older 3rd person blocking pose to be added and used on 1.7 and older servers.

**Comment by FlorianMichael** [comment]
Provide your VFP version as well as all installed mods

**Comment by AndrewScripter587** [comment]
Ok, my VFP version is 3.4.5, and here's a command prompt directory listing of my mods folder since i'm too lazy to actually write down a list: 
appleskin-fabric-mc1.21-3.0.5.jar 
authme-8.0.0+1.21.jar
BetterF3-11.0.2-Fabric-1.21.1.jar
BetterPingDisplay-Fabric-1.21.1-1.1.1.jar
chat_heads-0.13.0-fabric-1.21.jar
cloth-config-15.0.130-fabric.jar
fabric-api-0.104.0+1.21.1.jar
iris-fabric-1.8.0-beta.3+mc1.21.1.jar
justzoom_fabric_2.0.0_MC_1.21-1.21.1.jar        
konkrete_fabric_1.9.9_MC_1.21.jar
modelfix-1.21-1.6-fabric.jar                    (Model Gap Fix)
modmenu-11.0.2.jar
morechathistory-1.3.1.jar                       
MouseTweaks-fabric-mc1.21-2.26.jar
NBTac-FABRIC-1.21.1-1.3.4.jar             (NBT Autocomplete)      
placeholder-api-2.4.1+1.21.jar                  
skinlayers3d-fabric-1.6.7-mc1.21.jar
sodium-extra-fabric-0.6.0-beta.1+mc1.21.1.jar   
sodium-fabric-0.6.0-beta.1+mc1.21.jar
ViaFabricPlus-3.4.5.jar                         
waveycapes-fabric-1.4.9-mc1.21.jar

**Comment by FlorianMichael** [comment]
Can you try [this](https://ci.viaversion.com/view/Platforms/job/ViaFabricPlus/) version? I introduced some changes since the last release (3.4.5)

**Comment by AndrewScripter587** [comment]
The blocking pose is still the 1.8 pose on a 1.7 server when 1.7 is selected in VFP in the one in the link you provided. Here's some images for comparison (first 3 are vanilla 1.7's blocking pose and the 4th is the ViaFabricPlus blocking pose, which is 1.8's): 
![2024-09-15_15 20 47](https://github.com/user-attachments/assets/f6dd4aa7-42f6-430c-b278-8225ded77524)
![2024-09-15_15 20 50](https://github.com/user-attachments/assets/987f39c7-d491-4840-8850-781401eea279)
![2024-09-15_15 20 40](https://github.com/user-attachments/assets/1b996699-6c5c-4159-8736-babb51f97449)
![2024-09-15_15 23 33](https://github.com/user-attachments/assets/2ff88121-8bae-4389-9f99-1d4b7770dab7)

**Comment by lowercasebtw** [comment]
I was planning on working on this, but I haven't been able to convert the 1.7 code properly yet

**Comment by AndrewScripter587** [comment]
> I was planning on working on this, but I haven't been able to convert the 1.7 code properly yet

Makes sense, it is usually hard to port these kinds of things.

**Comment by lowercasebtw** [comment]
Yea, 1.9 changed how the positioning stuff worked and the same values don't work the same anymore, and requires re-working/finding what causes the difference in values

**Comment by AndrewScripter587** [comment]
Yeah, changing certain methods for stuff has its pros and cons (especially for a large game like Minecraft), like a pro may be making it easier to change and build on top of, but a con being something like this, where certain values don't work the same as they did anymore, making backwards/forwards compatibility, among other things, much harder to implement.

**Comment by lowercasebtw** [comment]
Hi, if you want accurate animations/item positions, you should use [Animatium](https://modrinth.com/mod/animatium/). 

Adding all the code to ViaFabricPlus to make this stuff accurate would be a lot of code to maintain/handle that might break without being noticed unless someone specifically kept track of it, and due to its nature probably shouldn't be added in VFP as it is a lot.

 I work on Animatium so I make sure its always up to date and functioning properly, Animatium was made originally to be used alongside ViaFabricPlus.
